pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.black]
+exclude = '''
+(
+ /(
+   docs
+   | node_modules
+   | build*.py
+   | media
+   | storage
+   | logs
+   | site-static
+   | static
+   | \.git
+   | \.npm
+   | \.tox
+  )/
+  | src/olympia/wsgi\.py
```

Comment:
😕 mmm?

Reply:
this was in Chris' patch. I meant to check why but I forgot obviously.